src/utils.js

```diff
@@ -0,0 +1,69 @@
+var _ = require('lodash');
+
+function format_address(address) {
+    if (isNoU(address) || address.length === 0) {
+        return '';
+    }
+
+    var len = address.length;
```

Comment:
Eep! Don't update lodash -- the version needs to match the one we have in
production (which the skeleton does). If padStart doesn't exist, maybe just
use pad instead.